.github/workflows/otp_pkg.yaml

```diff
@@ -86,9 +86,27 @@ jobs:
       - name: Deploy test
         if: ${{ steps.build.outcome == 'success' }}
         id: deploy-test
+        continue-on-error: true
         working-directory: robot
         run: |
           rebar3 grisp deploy
+      - name: Create GitHub Issue on Deploy Test Failure
+        if: ${{ steps.deploy-test.outcome == 'failure' }}
```

Comment:
Here you can handle both failures, if the IF also checks for the build step outcome.
Then you can remove the upper Create GitHub Issue and just leave this one.
The deploy step will not run in case of build failure as it also checks the outcome.